website/source/docs/secrets/ssh/index.html.md

```diff
@@ -288,6 +288,70 @@ username@<IP of remote host>:~$
 ```
 
 ----------------------------------------------------
+## III. CA Key Type
+
+When using this type, an SSH key is generated and then used to sign other SSH keys.
+The public half of the key is distributed to remote hosts while the private part
+stays within Vault. This allows SSH public keys to be signed by Vault and then
+verified using only the public key.
+
+### Configure a CA certificate
+
+To configure the backend, an SSH key pair must be generated to serve as the CA key:
+
+```text
+$ ssh-keygen -N '' -f ./ca -C 'SSH CA key pair'
+```
+
+Now that we have the SSH CA key pair, these can be used to configure the SSH CA backend:
+
+```text
+$ cat ca | vault write sshca/config/ca private_key=- public_key="$(cat ca.pub | tr -d '\n')"
```

Comment:
All references to the mount as `/sshca` should be changed back to `/ssh`. Users might be following the instructions with an existing mount at `/ssh`.